server/src/test/java/org/apache/druid/server/http/CoordinatorDynamicConfigTest.java

```diff
@@ -389,7 +639,27 @@ public void testUpdate()
     Assert.assertEquals(
         current,
         new CoordinatorDynamicConfig
-            .Builder(null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null)
+            .Builder(
```

Comment:
Nit: Maybe move `.Builder(` to the previous line because indentation looks a little off.